docs/getting_started/tutorial.md

```diff
@@ -211,13 +211,13 @@ if the tags aren't loaded.
 ## A basic blog
 
 We are now ready to create a blog. To do so, run
-`python manage.py startapp blog` to create a new app in your Wagtail site.
+`python manage.py startapp blog` to create a new app on your Wagtail site.
```

Comment:
"on your Wagtail site" would be more natural if we were talking about an area of a website that someone can visit, but in this case it's referring to the codebase. So maybe "project" is more accurate than "site"...

```suggestion
`python manage.py startapp blog` to create a new app in your Wagtail project.
```